### PR TITLE
[case-wip](Vault) Add basic vault case for P0

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/vaults/default/default.groovy [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        """
+    } catch (Exception e) {
+    }
+
+    try {
+        sql """
+            set $ as default vault
+        """
+    } catch (Exception e) {
+    }
```
Comment (924060929) [inline regression-test/suites/vaults/default/default.groovy:132]: You can use for refine test
```groovy
try_sql "set $ as default vault"`
```

**Comment by dataroaring** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Review by pingchunzhang — APPROVED** [review]
LGMT

**Comment by ByteYue** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**